src/sentry/rules/processing/delayed_processing.py

```diff
@@ -552,6 +554,10 @@ def cleanup_redis_buffer(
         filters["batch_key"] = batch_key
 
     buffer.backend.delete_hash(model=Project, filters=filters, fields=hashes_to_delete)
+    logger.info(
```

Comment:
should we put this under the feature flag so we don't get spammed by logs?

`if features.has("projects:num-events-issue-debugging", project)`